### PR TITLE
Add exec with error output processing

### LINKED ISSUE #9180: Provide logs from bootstrapping exec into ws pod
If workspace bootstrapping failed due to the fact that the initial exec with curl `$binary` failed due to connectivity issues of endpoint being malformed, there are no clues in logs. Che server just fails to start a workspace by timeout. The only way to find out what's happening is getting a shell into a workspace pod, and manually curl or wget binaries. This takes time.

There should be command syntax and output from this exec in case it returns failed response code.

### PR DESCRIPTION
**Pull request by akorneta**
### What does this PR do?
Adds bootstrapper command error stream processing, in cases when error stream is not empty, stops the bootstrapping and propagates appropriate logs.

### What issues does this PR fix or reference?
fixes #9180

### CODE REVIEW

**Comment by akorneta** [comment]
ci-build

**Comment by akorneta** [comment]
ci-test

File: infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesPods.java [hunk 1]
```diff
@@ -361,6 +370,57 @@ void stopWatch() {
...
   }
 
+  /**
+   * Executes command in specified container.
+   *
+   * @param podName pod name where command will be executed
+   * @param containerName container name where command will be executed
+   * @param timeoutMin timeout to wait until process will be done
+   * @param command command to execute
+   * @param outputConsumer command output callback
```
Comment (sleshchenko) [inline infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesPods.java]: Please document what are arguments of a consumer.

**Review by sleshchenko — APPROVED** [review]
LGTM

**Review by skabashnyuk — APPROVED** [review]

File: infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesMachine.java [hunk 2]
```diff
@@ -102,6 +103,11 @@ public void setServerStatus(String serverRef, ServerStatus status) {
     server.setStatus(status);
   }
 
+  public void exec(BiConsumer<String, String> outputConsumer, String... command)
```
Comment (garagatyi) [inline infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/KubernetesMachine.java]: Please,  add javadocs that would explain how the method and biconsumer is used

**Review by garagatyi — APPROVED** [review]

**Pull request merged**